packages/rich-text/src/create.js

```diff
@@ -113,6 +113,7 @@ function toFormat( { type, attributes } ) {
 	};
 }
 
+/* eslint-disable jsdoc/check-line-alignment */
```

Comment:
This is the only case where I disabled the rule because adding an @example tag would add a Usage mark in the readme.

Reply:
What if we modify the rule to skip formatting if we encounter a markdown code block? So that we don't have to use `@example` whenever we want to insert some formatted code.

Reply:
> Do you think it's really important and we should hold on until we have a solution for that? Or since it's not many cases we could go ahead with the current workaround?

No, I don't think it's that important. Just tossing some ideas for potential improvements :)